### PR TITLE
Add Governor DAO (GDAO) (ERC20)

### PR DESCRIPTION
**Pull request by 0xJonHoldsCrypto**
Adding info for ERC20 token 'Governor DAO (GDAO)'.

PR created by <a href="https://assets.trustwallet.com">TW Assets Management</a>

### CODE REVIEW

File: blockchains/ethereum/assets/0x515d7E9D75E2b76DB60F8a051Cd890eBa23286Bc/info.json [hunk 1]
```diff
@@ -0,0 +1,69 @@
+{
+    "name": "Governor DAO (GDAO)",
+    "type": "ERC20",
+    "symbol": "GDAO",
+    "decimals": 18,
+    "website": "https://governordao.org",
+    "description": "Governor DAO is a Wyoming-based Decentralized Autonomous Organization positioned as the “DAO of DAOs”. Governors offer a suite of products and services for projects looking to build out DAO qualities in their own communities. Our offerings include an industry-first sybil-resistance product for one-voice-one-vote governance, as well as governance bootstraps for new communities, consultations, and smart contract porting.\n",
+    "explorer": "https://etherscan.io/token//0x515d7e9d75e2b76db60f8a051cd890eba23286bc",
```
Comment (0xJonHoldsCrypto) [inline blockchains/ethereum/assets/0x515d7E9D75E2b76DB60F8a051Cd890eBa23286Bc/info.json:8]: https://etherscan.io/token/0x515d7E9D75E2b76DB60F8a051Cd890eBa23286Bc

**Review by 0xJonHoldsCrypto — COMMENTED** [review]
https://etherscan.io/token/0x515d7E9D75E2b76DB60F8a051Cd890eBa23286Bc

File: blockchains/ethereum/assets/0x515d7E9D75E2b76DB60F8a051Cd890eBa23286Bc/info.json [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+            "name": "docs",
+            "url": "https://docs.governordao.org"
+        },
+        {
+            "name": "forum",
+            "url": "https://gdao.info"
+        },
+        {
+            "name": "discord",
+            "url": "https://discord.gg/MVh5NkN7gt"
```
Comment (0xJonHoldsCrypto) [inline blockchains/ethereum/assets/0x515d7E9D75E2b76DB60F8a051Cd890eBa23286Bc/info.json:42]: there is no current possible discord.com linking strategy - all links end in .gg

**Comment by 0xJonHoldsCrypto** [comment]
Is the Discord formatting error just a false pos? There is no current Discord.com linking strategy that we are aware (or Discord support for that matter) of. 

Also, do we have to pay again to add our logo and updated links?

thanks!

**Comment by cryptomanz** [comment]
As per our policy, brand new tokens are not included, there is a minimum requirement of 25,000 holder addresses and 50,000 transactions, in order to filter out scam tokens.
See the criteria at : https://developer.trustwallet.com/assets/requirements .

**Comment by 0xJonHoldsCrypto** [comment]
we're over a year old and in the top 200 - those are hard numbers?

**Pull request closed without merging**